pr_agent/algo/pr_processing.py

```diff
@@ -19,26 +20,33 @@
 PATCH_EXTRA_LINES = 3
 
 
-def get_pr_diff(git_provider: Union[GithubProvider, Any], token_handler: TokenHandler) -> str:
+def get_pr_diff(git_provider: Union[GithubProvider, Any], token_handler: TokenHandler,
+                add_line_numbers_to_hunks: bool = False, disable_extra_lines: bool =False) -> str:
```

Comment:
**Suggestion:** Consider using a class or a data structure to encapsulate the parameters of the `get_pr_diff` function. This will make the function signature cleaner and easier to manage as the number of parameters grows.
```suggestion
class PrDiffParams:
    def __init__(self, git_provider, token_handler, add_line_numbers_to_hunks=False, disable_extra_lines=False):
        self.git_provider = git_provider
        self.token_handler = token_handler
        self.add_line_numbers_to_hunks = add_line_numbers_to_hunks
        self.disable_extra_lines = disable_extra_lines

def get_pr_diff(params: PrDiffParams) -> str:
```